src/main/java/at/hannibal2/skyhanni/config/features/inventory/chocolatefactory/ChocolateFactoryRabbitWarningConfig.java

```diff
@@ -34,6 +36,34 @@ public class ChocolateFactoryRabbitWarningConfig {
     @ConfigEditorSlider(minValue = 1, maxValue = 20, minStep = 1)
     public int repeatSound = 20;
 
+    @Expose
+    @ConfigOption(name = "Flash Screen", desc = "Choose the stray rabbit type to flash the screen for.")
+    @ConfigEditorDropdown
+    public FlashScreenTypeEntry flashScreenType = FlashScreenTypeEntry.SPECIAL;
+
+    public enum FlashScreenTypeEntry {
+        REGULAR("Regular"),
```

Comment:
"regular only" is not needed. rather have the options be "only special", "all", "none". or something similar